litmus-portal/frontend/src/views/ChaosWorkflows/BrowseAnalytics/WorkflowRunsBarChart/index.tsx

```diff
@@ -0,0 +1,396 @@
+/* eslint-disable max-len */
+/* eslint-disable no-console */
+import React, { useEffect } from 'react';
+import Plotly from 'plotly.js';
+import createPlotlyComponent from 'react-plotly.js/factory';
+import moment from 'moment';
+
+const Plot = createPlotlyComponent(Plotly);
+
+interface SelectedWorkflowRunData {
+  testsPassed: number;
+  testsFailed: number;
+  resilienceScore: number;
+  testDate: string;
+  xLoc: number;
+  yLoc: number;
+  workflowRunID: string;
+}
+
+export interface PopOverCallBackType {
+  (selectedWorkflowRunData: SelectedWorkflowRunData, visible: boolean): void;
+}
+
+interface WorkflowRunData {
+  testsPassed: number;
+  testsFailed: number;
+  resilienceScore: number;
+  testDate: string;
+  workflowRunID: string;
+}
+
+interface WorkflowRunsBarChartProps {
+  workflowRunData: WorkflowRunData[];
+  workflowID: string;
+  callBackToShowPopOver: PopOverCallBackType;
+}
+
+const WorkflowRunsBarChart: React.FC<WorkflowRunsBarChartProps> = ({
+  workflowRunData,
+  callBackToShowPopOver,
+  workflowID,
+}) => {
+  const [plotData, setPlotData] = React.useState<any[]>([]);
+
+  const [plotLayout, setPlotLayout] = React.useState<any>({});
+
+  const [visible, setVisible] = React.useState<boolean>(false);
```

Comment:
Remove the any type

Reply:
This cant be done as i will have to type all options plotly can provide. @S-ayanide